extensions/podman/packages/extension/src/utils/powershell.ts

```diff
@@ -17,6 +17,8 @@
  ***********************************************************************/
 import * as extensionApi from '@podman-desktop/api';
 
+import { getTelemetryLogger } from '../extension';
```

Comment:
I think having this kind of dependency is an anti-pattern

sub-code should not import something from the extension

extension is importing the sub-code